src/main/java/at/hannibal2/skyhanni/config/features/garden/laneswitch/LaneswitchFarmConfig.java

```diff
@@ -0,0 +1,52 @@
+package at.hannibal2.skyhanni.config.features.garden.laneswitch;
+
+import at.hannibal2.skyhanni.config.HasLegacyId;
+import com.google.gson.annotations.Expose;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorDropdown;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorSlider;
+import io.github.moulberry.moulconfig.annotations.ConfigOption;
+
+public class LaneswitchFarmConfig {
+    @Expose
+    @ConfigOption(name = "Amount of Plots", desc = "The amount of plots the current Farm has.")
```

Comment:
I don't think "Farm" should be upper case. Also maybe something like "Lane Length" and "How many plots each lane is long" might be better? I'm not sure.